### ISSUE TITLE
Improve search results page a11y

### ISSUE THREAD

**Issue by richardbporter** [uiowa/uiowa#2021]
# Problem 

The search results page has misleading colors that creates confusion over which link to click on. 

> I was using the search function on some of the sites that are using the new UIDS and noticed something odd. 
When you do any type of search, (take this https://uiowa.edu/google-search?search=search), search results are presented in a title, line break, URL, line break description format. However, the URL, despite being highlighted in a different color, isn't a hyperlink—only the page title is, which happens to be in black text with no underline.
This felt like an accessibility red flag and just a counterintuitive user experience. Just wanted to share this feedback.

<img width="870" alt="Screen Shot 2020-12-21 at 6 14 41 AM" src="https://user-images.githubusercontent.com/1036433/102776365-3c205d80-4354-11eb-9a5f-ab26a7f132dd.png">

# Proposed Solution

- [x] Adjust https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/search-results.scss and switch color treatment between Title and URL and add hover underline to the title as well. 

<img width="908" alt="Screen Shot 2020-12-21 at 6 15 21 AM" src="https://user-images.githubusercontent.com/1036433/102776452-5ce8b300-4354-11eb-87ef-baff0af5c3c4.png">

**Comment by joewhitsitt** [comment]
This got merged.
